lib/GeoIP2/Record/Location.pm

```diff
@@ -15,6 +15,18 @@ has accuracy_radius => (
     predicate => 'has_accuracy_radius',
 );
 
+has average_income => (
+    is        => 'ro',
+    isa       => NonNegativeInt,
+    predicate => 'has_average_income',
+);
+
+has estimated_population => (
+    is        => 'ro',
+    isa       => NonNegativeInt,
```

Comment:
One day our datasources could theoretically have valid `0` values (for example an area that's populated by people who don't work, or an area where people only work but no one actually lives there.)  Let's leave it as it is.